### DISCUSSION TITLE
flameshot support under wayland

### DISCUSSION THREAD

**Discussion by ayham-1** [qtile/qtile#4392] [Ideas]
Flameshot is arguably best screenshot tool on linux.

How to implement flameshot support for wayland qtile?

From my understanding individual wayland backends need to accommodate for it somehow?

I am willing to work on the feature, where should I start?

Maybe there is a quick fix available?

**Comment by ervinpopescu** [comment]
As qtile uses pywlroots, I assume you can try [this](https://github.com/flameshot-org/flameshot/blob/master/docs/Sway%20and%20wlroots%20support.md), as a starting point.
